modules/administration-guide/pages/configuring-workspace-target-namespace.adoc

```diff
@@ -0,0 +1,62 @@
+:_content-type: PROCEDURE
+:navtitle: Configuring {orch-namespace} name
+:keywords: administration guide, configuring, namespace
+:page-aliases: 
+
+[id="configuring-user-{orch-namespace}-name-for-automatic-provisioning_{context}"]
+= Configuring user {orch-namespace} name for automatic provisioning
+
+You can configure the {orch-namespace} name template that {prod-short} is using to create the required {orch-namespace} when starting a workspace.
+
+A valid {orch-namespace} name template follows this convention:
+
+* The `<username>` or `<userid>` placeholder is mandatory.
+
+* {prod-short} reduces usernames or IDs incompatible with {orch-name} objects naming convention to the nearest valid name by replacing groups of unsuitable symbols with the `-` symbol. 
+
+* {prod-short} evaluates the `<userid>` placeholder into a 14 character long string, and adds a random 6 character long suffix to prevents IDs from collisions. The result is stored in the user preferences for reuse.
```

Comment:
```suggestion
* {prod-short} evaluates the `<userid>` placeholder into a 14 character long string, and adds a random six character long suffix to prevent IDs from colliding. The result is stored in the user preferences for reuse.
```
- Per ISG: `In text, use words for zero through nine. In text, use numerals for 10 and greater.`